DigitalLearningSolutions.Data/DigitalLearningSolutions.Data.csproj

```diff
@@ -20,5 +20,9 @@
     <PackageReference Include="System.Drawing.Common" Version="5.0.2" />
   </ItemGroup>
 
+  <ItemGroup>
+    <Folder Include="ViewModels\UserCentreAccount\" />

```

Comment:
what does this do, @sarathlal-sarangadharan?

Reply:
@kevwhitt-hee It was intend to to under the folder "ViewModels\SuperAdmin\Users". At first created a new folder and missed to move. Can I move to expected folder?

Reply:
Moved to  "ViewModels\SuperAdmin\Users" and unused folder deleted. Please review.

Reply:
Can we revert this change to the Data.csproj file? I don't think it is needed.